### DISCUSSION TITLE
这个alist地址总是挂载不成，有高手帮看一下吗

### DISCUSSION THREAD

**Discussion by zhaodefang** [AlistGo/alist#3065] [Q&A]
https://138.2.112.184/
![image](https://user-images.githubusercontent.com/3375652/212638590-0ab32cb2-48af-4cca-90c8-e21fe09214c0.png)
这是个443端口的，好象是需要证书，可是配置里不知道怎么弄

**Comment by NovaShen555** [comment]
你发Q&A我告诉你咋搞

**Reply by NovaShen555** [reply]
兄弟你看他证书其实就知道他域名就是alist.haimian.eu.org啊
你挂载https://alist.haimian.eu.org/就好啦
记得采纳

**Reply by NovaShen555** [reply]
人呢
